common/display/displayqueue.cpp

```diff
@@ -490,49 +496,11 @@ void DisplayQueue::SetColorCorrection(struct gamma_colors gamma,
   free(lut);
 }
 
-bool DisplayQueue::GetBroadcastRGBProperty(int fd, uint32_t id, uint32_t type,
-                                           const char* name, uint32_t* p_id,
-                                           uint64_t* p_value,
-                                           drmModePropertyPtr* p) {
-  drmModePropertyPtr p_t;
-  drmModeObjectPropertiesPtr props;
-  bool ret = false;
-  int i;
-
-  props = drmModeObjectGetProperties(fd, id, type);
-  for (i = 0; i < props->count_props; i++) {
-    p_t = drmModeGetProperty(fd, props->props[i]);
-    if (!p_t)
-      continue;
-
-    if (strcmp(p_t->name, name) == 0) {
-      ret = true;
-      if (p_id)
-        *p_id = props->props[i];
-      if (p_value)
-        *p_value = props->prop_values[i];
-      if (p)
-        *p = p_t;
-      else
-        drmModeFreeProperty(p_t);
-
-      break;
-    }
-    drmModeFreeProperty(p_t);
-  }
-
-  drmModeFreeObjectProperties(props);
-  return ret;
-}
-
 bool DisplayQueue::SetBroadcastRGB(const char* range_property) {
-  drmModePropertyPtr p;
-  int32_t p_value = -1;
-  uint32_t p_id;
+  uint64_t p_value = -1;
+
+  drmModePropertyPtr p = drmModeGetProperty(gpu_fd_, broadcastrgb_id_);
```

Comment:
yes, i think so, as we need map drm_mode_property_enum's name to value so that we can use name "Full" to set value